ws.py

```diff
@@ -197,6 +197,10 @@ def check_socket_connections():
         "room": GlobalVars.wrap.get_room("388"),
         "unwantedReasons": []
     },
+    {   "sites": ["meta.stackexchange.com"],
```

Comment:
`flake8` dislikes this, but the fix should be trivial -- just add a newline after the opening brace and reindent.